libraries/chain/protocol_feature_manager.cpp

```diff
@@ -65,7 +65,7 @@ Also allows a contract to send a deferred transaction in a manner that enables t
          } )
          (  builtin_protocol_feature_t::fix_linkauth_restriction, builtin_protocol_feature_spec{
             "FIX_LINKAUTH_RESTRICTION",
-            fc::variant("a98241c83511dc86c857221b9372b4aa7cea3aaebc567a48604e1d3db3557050").as<digest_type>(),
+            fc::variant("e9cc714eb3f294a5c3a62f0b6b5f81a0000d9de75fd652d68da1c7fb81fccaf9").as<digest_type>(),
```

Comment:
Do we know why these 2 protocols, and only these protocols, were changed?

Reply:
The message being hashed changed (eosio -> sysio)

Reply:
Sorry, when I wrote these up, I forgot that the diff I am looking about isn't the diff with the upstream leap branch.